rusk-abi/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.3.0] - 2021-03-16
+
+### Changed
+
+- Updated `verify_proof` to accept verifier data [#247]
+
```

Comment:
Shouldn't we do a pre-release for that? Or leave this on `Unreleased` for now?

Reply:
This is correct.
@vlopes11, create from `master` a branch called `rusk-abi-0.3`, as we did for `0.2`, and make sure this PR points to it instead of `master`.
The `Changed` section should be therefore under `Unreleased` as @CPerezz said, and no additional version are added yet.

Plus, the comment should use present tense (see: https://chris.beams.io/posts/git-commit/), and whenever is possible use the verbs of the CHANGELOG's section (So: Add, Remove, Change, Fix). In some cases we will add some aliases so "Update" or "Replace" will be fine too, however in this case it should be better something like:
```
- Change `verify_proof` to take `VerifierData` instead of `VerifierKey` and PI positions
```